types/2019-12-03/Charges.d.ts

```diff
@@ -926,12 +926,7 @@ declare module 'stripe' {
 
         interface Fpx {
           /**
-           * Account holder type, if provided. Can be one of `individual` or `company`.
-           */
-          account_holder_type: Fpx.AccountHolderType | null;
```

Comment:
This is a breaking change, I don't understand what caused this and I haven't seen any PR cause this